subsys/bluetooth/host/conn.c

```diff
@@ -1007,6 +1007,12 @@ void bt_conn_security_changed(struct bt_conn *conn, enum bt_security_err err)
 			cb->security_changed(conn, conn->sec_level, err);
 		}
 	}
+#if (defined(CONFIG_BT_KEYS_OVERWRITE_OLDEST))
+	if (!err &&
+		(conn->sec_level >= BT_SECURITY_L2)) {
```

Comment:
This line is not correctly aligned with the opening `(` on the first line. You also don't need the `()`. The wrong alignment makes this look super confusing since the code inside the branch is now using the same indentation, i.e. it's hard to see what's part of the condition and what's part of the branch.